css/style.css

```diff
@@ -16,4 +17,175 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+.container {
+    margin: 0 auto;
+}
+
+h2, h3 {
+    font-weight: 300;
+
+}
+/*Header*/
+
+ header {
+    display: flex;  
+}
+
+.logo {
+    width: 2em;
+    padding-left: 2em;
+}
+
+nav {
+    display: flex;
```

Comment:
```suggestion
```
The `nav` element only has one child, and we don't need to use flexbox on it so we can remove this. Usually, we remove any code that doesn't have an effect